std/js/Cookie.hx

```diff
@@ -41,6 +41,12 @@ class Cookie {
 		if (domain != null) {
 			s += ";domain=" + domain;
 		}
+		if (secure != null) {
```

Comment:
You are adding secure even if it's set to false.

Reply:
oops! fixed. Thanks!